src/trackedit/api/projectapi.cpp

```
@@ -0,0 +1,126 @@
+}
+
+ITrackeditProjectPtr ProjectApi::trackeditProject() const
+{
+    return globalContext() ? globalContext()->currentTrackeditProject() : nullptr;
+}
+
+Clips ProjectApi::sortedClips(int trackIndex) const
+{
+    auto prj = trackeditProject();
+    if (!prj) {
+        return {};
```

Cppcheck (2.20.0)

[style] The function 'fileName' is never used.

(unusedFunction)